shapely/ops.py

```diff
@@ -533,7 +533,7 @@ def substring(geom, start_dist, end_dist, normalized=False):
         elif current_distance >= max_dist:
             break
 
-        current_distance += LineString([p1, p2]).length
+        current_distance += ((p2.x - p1.x) ** 2 + (p2.y - p1.y) ** 2) ** 0.5
```

Comment:
@kylebarron p1 and p2 are tuples, (x, y), not instances of Point. That's what our CI is protesting about.

Reply:
Oops, you're right. I hadn't noticed the CI fail.